projects/packages/lazy-images/src/js/intersectionobserver-polyfill.js

```diff
@@ -23,17 +23,17 @@
 		'IntersectionObserverEntry' in window &&
```

Comment:
Now that you fixed .eslintignore, you should be able to drop the changes to this file.